feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -310,6 +310,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		}
 	}
 
+	nc1InputWeight := uint64(200)
+	if deviations.SchedulerInputWeightLimit(ondatra.DUT(t, "dut")) {
```

Comment:
No need to lookup the DUT when you already have a reference to it. Just "if deviations.SchedulerInputWeightLimit(dut) {" works.

Reply:
addressed